ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -934,13 +934,29 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
     render() {
       const {enableBillingUpgrade} = serverConfigStore.getValue();
       const {
-        populationChecked,
```

Comment:
none of these changes are necessary to the PR.  I can revert if you don't like them.